apps/login/Guest/pupa.inc

```diff
@@ -12,3 +12,9 @@
 
 ;terminal prompt and scroll back
 (defq *env_terminal_prompt* (cat *env_user* ">") *env_terminal_lines* (* 40 10))
+
+;default text files to open when edit opens
+(defq *env_edit_auto* '("apps/edit/message"))
+
+;ensure that variables are imported before being called.
+(task-sleep 10000)
```

Comment:
You should not need to do this ! So this warrants further investigation, possible on my part once this gets merged.

I'll let you think on the comments here and in the "Textfield and Editor" issue, and when your happy I'll merge over and do some digging !

Reply:
Maybe something I did. I'll remove it and see how things are working now.